### PR TITLE
Add boundaries for time in months

### LINKED ISSUE #49: Add time boundaries to budget & transactions display.
Blocked by #45

### PR DESCRIPTION
**Pull request by WoobieApps**
closes #49

### CODE REVIEW

File: app/routes/routes.py [hunk 1]
```diff
@@ -34,8 +34,8 @@
 
 
 this_date = datetime.now().strftime("%B, %Y")
-this_month = date.today().month
-this_year = date.today().year
+this_month = date.today().month  # TODO: make this changeable in nav
```
Comment (JohnnyBeet) [inline app/routes/routes.py:37]: tbh i think it's better to move it to __init__.py

**Review by JohnnyBeet — CHANGES_REQUESTED** [review]
One small thing, either persuade me out of it or change it and after that you can merge.

File: app/routes/routes.py (hunk 1 above)
Comment (WoobieApps) [reply app/routes/routes.py:37]: __init__.py is called once when a package is imported, we want to have those variables available constatly when app is running. If You move it to __init__.py, You will lose it as soon as this script finishes. Correct me if I am wrong.

Comment (JohnnyBeet) [reply app/routes/routes.py:37]: True, if those were constant then it would work, but those are not constant. My bad, you can merge.

**Review by JohnnyBeet — APPROVED** [review]

**Pull request merged**